### PR TITLE
Quest Port from E8

### PR DESCRIPTION
**Pull request by Kanzaji**
*This pull request will probably be here for at least week 😅 Also a lot of this what I wrote here could also change :D*

For now only port and fix for E8 quests, later on will be working on adding more quest lines, adding those that are hidden at the moment and other fixes :D

**List of quest lines I'm going to fix here:**

- Tools and Armor / Storage / Automation (Technically "fix", as there is a lot of mods missing)
- IE -> Fixed
- Mekanism -> Fixed
- Nature's Aura -> Fixed
- Occultism -> Fixed
- Powah! -> Fixed
- Create -> Fixed, just missing PonderJS for now

**Quest lines that will be done on other times**
- AE2 -> This.. will take some time. Going to do another PR when its done.
- PNC -> Fixed, but Motoko is going to rework this one
- Ars Noveau -> I... I'm leaving this one alone ^^" Also Motoko will rework this one so 😋

**Quest lines hidden because mod is missing / WIP**

- RFTools -- mod is kinda WIP
- Tcon
- Thermal
- BM
- Botania
~~- RS~~
~~- ID~~
~~- Tetra~~

~~Still not sure which of mods hidden are meant to be in E9, so I'm not touching them for now 😅~~
Also I'm not sure which mods are worth their own quest lines, as there is few new ones, but this will come with time as well :D

Added mods: Powah!, Tiab, ~~Coocking for blockheads~~, Redstone Pen, ~~Ender Storage~~, Plonk, Macaws bridges, Applied Mekanisticsm, Entangled, Engineers decor

### CODE REVIEW

**Comment by Kanzaji** [comment]
Building was "fixed", a lot of mods aren't in E9 yet or will not be, and for now I'm fixing quests, 😅
2 are "missing", just because 2 of Macaw's mods doesn't have port to 1.19.2 yet, and I'm guessing those will be added in the future.

obraz

**Comment by Kanzaji** [comment]
Okay looking at how much mods are missing, I need to halt fix / rework of "Explore Enigmatica" section 😅 
Literally Automation section is empty, so there is no sense atm to try to fix those.

**Comment by Gcat101** [comment]
No mods OmegaLUL

**Comment by NielsPilgaard** [comment]
> **Quest lines hidden because mod is missing / WIP**
>
>- RFTools -- WIP
>- Tetra
>- Tcon
>- ID
>- RS
>- Create
>- Powah!
>- Thermal
>- BM
>- Botania
>
>Still not sure which of mods hidden are meant to be in E9, so I'm not touching them for now 😅
>Also I'm not sure which mods are worth their own quest lines, as there is few new ones, but this will come with time as well :D

Of those mods, the following won't be in E9:

- RFTools Power
- Integrated Dynamics 
- Refined Storage (I think, not 100% there)
- Tetra

**Comment by Kanzaji** [comment]
I'm gonna still wait with deletion of Refined Storage, as it is not 100% goner ^^"

**Comment by Kanzaji** [comment]
PS: I could take a bit longer to actually port those quests, I don't want to do them asap, to make mistakes and left broken ones. I have a lot of work recently in real life, so it makes it difficult to make good job on quests 😅
I'm sorry for that, but I'm just not able to work much on them except weekends from not my reasons.

**Comment by NielsPilgaard** [comment]
> PS: I could take a bit longer to actually port those quests, I don't want to do them asap, to make mistakes and left broken ones. I have a lot of work recently in real life, so it makes it difficult to make good job on quests 😅 I'm sorry for that, but I'm just not able to work much on them except weekends.

All good ❤️

**Comment by MuteTiefling** [comment]
I'd suggest we just go ahead and merge this at this point so that I can more easily assist with quest changes. Tackle by mod instead of one monolithic PR.

**Comment by Kanzaji** [comment]
I have finally some free time in another days ^^" so I will be working on these. Give me like 3/4 days ^-^"

**Comment by MuteTiefling** [comment]
Just the same, getting them in where we can both work on them as time permits would be for the best. I'm also fearful of making any new quests for fear of accidental file conflicts.

**Comment by Kanzaji** [comment]
Well alrighty! Gonna just do another PR tmr when I will get something done ^^

**Comment by Kanzaji** [comment]
Okay so.. it was much less work than I thought initially ^^" One free day where I decided to spent it entirely on quests looks like was enough to fix up most of the quests (and actually more mods that ported helped as well)
Anyhow I'm ready for review :D

**Comment by NielsPilgaard** [comment]
I'd prefer if you left out schematics and building gadget patterns from this PR :)

File: kubejs/server_scripts/base/recipes/eccentrictome/shapeless.js [hunk 1]
```diff
@@ -63,13 +69,20 @@ ServerEvents.recipes((event) => {
...
+                    },
                     rftoolsbase: {
                         0: {
                             Count: 1,
                             id: 'rftoolsbase:manual'
                         }
                     }
-                }
+                },
+                'eccentrictome:version': 1
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/base/recipes/eccentrictome/shapeless.js:85]: This part scares me. @MuteTiefling why'd you omit it the first time around, any particular reason?

Comment (MuteTiefling) [reply kubejs/server_scripts/base/recipes/eccentrictome/shapeless.js:85]: Wasn't intentional. It just didn't show up in the kjs_hand output.

Comment (Kanzaji) [reply kubejs/server_scripts/base/recipes/eccentrictome/shapeless.js:85]: Older recipe worked fine ^^" It just showed up for me with kjs_hand so I added it.

**Review by MuteTiefling — APPROVED** [review]
We should probably exclude the moose headgear and night vision goggle artwork considering the base mods those come from are not in the pack. 

The rest looks good. I will need to see the quests in game to really get a feel for the quests

**Comment by MuteTiefling** [comment]
Gonna go ahead and merge. Will yoink the artwork out after.

**Pull request merged**